pkg/storaged/client.js

```diff
@@ -425,11 +426,32 @@
                 });
         }
 
+        function enable_nfs_features() {
+            return cockpit.spawn([ "which", "mount.nfs" ], { err: "ignore" }).then(
+                function () {
+                    client.features.nfs = true;
+                    client.nfs.start();
+                    return cockpit.resolve();
+                },
+                function () {
+                    return PK.call("/org/freedesktop/PackageKit", "org.freedesktop.DBus.Properties",
+                                   "Get", [ "org.freedesktop.PackageKit", "VersionMajor" ]).then(
```

Comment:
This is also a good candidate for moving into lib/packagekit. (Not a blocker for this PR)

Reply:
It has been moved.